node/pkg/driver/node_utils_test.go

```diff
@@ -419,3 +423,219 @@ func TestGetVolumeUuid(t *testing.T) {
 	}
 
 }
+
+func TestGetFileSystemVolumeStats(t *testing.T) {
+	dir1, err := ioutil.TempDir("", "dir_1")
+	if err != nil {
+		t.Fatalf("TestGetFileSystemVolumeStats failed: %s", err.Error())
+	}
+	defer os.RemoveAll(dir1)
+
+	tmpfile1, err := ioutil.TempFile(dir1, "test")
```

Comment:
I think it would be better to mock the file system, but this addition can be in a separate PR

Reply:
done